op-acceptance-tests/tests/base/chain/unsafe_sync_test.go

```diff
@@ -0,0 +1,86 @@
+package chain
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/ethereum-optimism/optimism/op-devstack/devtest"
+	"github.com/ethereum-optimism/optimism/op-devstack/presets"
+	"github.com/ethereum-optimism/optimism/op-supervisor/supervisor/types"
+	"github.com/stretchr/testify/require"
+)
+
+// TestUnsafeSync tests that the verifier nodes are able to sync the unsafe head of the chain.
+// It does this by waiting for the sequencer to produce 10 unsafe blocks and requiring the verifier nodes to
```

Comment:
Comment says 10 but code says 3. Perhaps change this to (something like)?

```suggestion
// It does this by waiting for the sequencer to produce NUM_UNSAFE_BLOCKS unsafe blocks and requiring the verifier nodes to
```

Reply:
I think I wanted it to be 10 but was debugging with 3. But good catch!